build.gradle

```diff
@@ -21,7 +22,7 @@ ext {
     http_version = '4.5.13' // Used by integration tests
 }
 
-subprojects {
+allprojects {
```

Comment:
This is done because the Xenit conventions gradle plugins need a **group** and **version** set on root-level.